### DISCUSSION TITLE
Use in multi level

### DISCUSSION THREAD

**Discussion by Mohammad99k** [spatie/laravel-permission#1812] [Q&A]
Hi
Can I use this package for multiple level?
For example I have level 1 and level 2 and each level have multiple role or permission for example level 1 have a manager and level 2 have another manager.

**Comment by erikn69** [comment]
Hi @Mohammad99k 

I made a pull request for something like that, #1804 ([usage](https://github.com/erikn69/laravel-permission/blob/teams_feature/docs/basic-usage/teams-permissions.md))
Maybe you can test it, or meanwhile replace relations from the traits on your model with the same idea, I don't know if it will be merged, but your review can be helpful

Also duplicate of #1782

**Reply by Mohammad99k** [reply]
Thank you
